src/main/java/appeng/integration/modules/ic2/IC2Module.java

```diff
@@ -46,22 +46,20 @@ public IC2Module()
 	public void postInit()
 	{
 		final IP2PTunnelRegistry reg = AEApi.instance().registries().p2pTunnel();
-		reg.addNewAttunement( this.getItem( "copperCableItem" ), TunnelType.IC2_POWER );
-		reg.addNewAttunement( this.getItem( "insulatedCopperCableItem" ), TunnelType.IC2_POWER );
-		reg.addNewAttunement( this.getItem( "goldCableItem" ), TunnelType.IC2_POWER );
-		reg.addNewAttunement( this.getItem( "insulatedGoldCableItem" ), TunnelType.IC2_POWER );
-		reg.addNewAttunement( this.getItem( "ironCableItem" ), TunnelType.IC2_POWER );
-		reg.addNewAttunement( this.getItem( "insulatedIronCableItem" ), TunnelType.IC2_POWER );
-		reg.addNewAttunement( this.getItem( "insulatedTinCableItem" ), TunnelType.IC2_POWER );
-		reg.addNewAttunement( this.getItem( "glassFiberCableItem" ), TunnelType.IC2_POWER );
-		reg.addNewAttunement( this.getItem( "tinCableItem" ), TunnelType.IC2_POWER );
-		reg.addNewAttunement( this.getItem( "detectorCableItem" ), TunnelType.IC2_POWER );
-		reg.addNewAttunement( this.getItem( "splitterCableItem" ), TunnelType.IC2_POWER );
+
+		reg.addNewAttunement( this.getItem( "cable", "type:copper" ), TunnelType.IC2_POWER );
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "cable" 8 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1192)